### PR TITLE
feat: points page

### PR DESCRIPTION
**Pull request by schmanu**
## What this PR changes
- Adds new page for points
- Shows total points gained for global activities
- Shows global leaderboard
- Shows global activities and campaign info

### CODE REVIEW

File: src/components/BoostCounter/index.tsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ const digitRotations: Record<number, number> = {
 const BoostCounter = ({
   value,
   direction,
+
```
Comment (compojoom) [inline src/components/BoostCounter/index.tsx:22]: prettier out of control?

File: src/config/constants.ts [hunk 2]
```diff
@@ -115,3 +115,8 @@ export const UNLIMITED_APPROVAL_AMOUNT = BigNumber.from(2).pow(256).sub(1)
 
 export const SEASON2_START = 160
 export const SEASON1_START = 33
+
+export const GLOBAL_CAMPAIGN_IDS: ChainConfig<string> = {
+  [Chains.SEPOLIA]: 'fa9f462b-8e8c-4122-aa41-2464e919b721',
+  [Chains.MAINNET]: 'fa9f462b-8e8c-4122-aa41-2464e919b721',
```
Comment (compojoom) [inline src/config/constants.ts:121]: same campaign ids on both chains?

**Review by compojoom — APPROVED** [review]

File: src/components/Points/ActivityPointsFeed.tsx [hunk 3]
```diff
@@ -16,21 +16,23 @@ const HiddenValue = () => (
 )
 
 export const ActivityPointsFeed = ({ campaign }: { campaign?: Campaign }) => {
-  const ownEntry = useOwnCampaignRank(campaign?.resourceId)
+  const { data: ownEntry, isLoading } = useOwnCampaignRank(campaign?.resourceId)
+
+  console.log(ownEntry, isLoading)
```
Comment (compojoom) [inline src/components/Points/ActivityPointsFeed.tsx:21]: you forgot to remove that.

File: src/config/constants.ts (hunk 2 above)
Comment (schmanu) [reply src/config/constants.ts:121]: Mainnet does not exist yet. Will be created momentarily and then replaced :)

**Pull request merged**